### ISSUE TITLE
Set first run time/date

### ISSUE THREAD

**Issue by thelastblt** [mattwebbio/orbital-sync#388]
### What feature would you like added?

I would like to specify a first run for the update. Currently orbitalsync runs on container creation at the specified interval. Because orbitslsync takes down the secondary pihole there could be network issues. 

Setting a first run time would allow for the updates to happen at a low network traffic time and could persist through container updates. 

### Version

1.8.1

**Comment by kylekrajnyak** [comment]
Agreed, would be nice to be able to set the first run time. In the meantime, you could set a systemd timer to restart the container every day at a set time, say 3am, to achieve the same end result.
